enthought_example/example_data_views/example_data_view.py

```diff
@@ -27,5 +29,7 @@ def plot_line(self):
         return plot
 
     def __plot_default(self):
-        self._plot = self.plot_line()
-        return self._plot
+        plot = self.plot_line()
+        self.plot_updater.start()
```

Comment:
Didn't you add a comment here explaining the rationale behind the `do_later` call?

Reply:
In the parent class (`wfmanager`), not here; should I add it here too?